src/nvim/window.c

```diff
@@ -5373,19 +5379,19 @@ int match_add(win_T *wp, char_u *grp, char_u *pat, int prio, int id, list_T *pos
   // Set up position matches
   if (pos_list != NULL)
   {
-    linenr_T	toplnum = 0;
-    linenr_T	botlnum = 0;
-    listitem_T	*li;
-    int		i;
+    linenr_T    toplnum = 0;
+    linenr_T    botlnum = 0;
+    listitem_T  *li;
+    int         i;
```

Comment:
We don't align variables like this. https://neovim.io/develop/style-guide.xml#Horizontal_Whitespace